_episodes/02-what-is-AI.md

```diff
@@ -13,73 +13,67 @@ keypoints:
 
 ## What does not exist?
 
-Siri and Alexa can answer simple instructions but use scripted interactions, there is not an understanding of the world like fictional intelligent robots. There is currently nothing close to Artificial General Intelligence, and we wouldn't know where to start in building it. The idea that a computer "understands" a problem is in a  narrow, technical sense, not the way a human understands. 
+Siri and Alexa can answer simple instructions using scripted interactions, but they lack what is known as Artificial General Intelligence (AGI) - an understanding of the world that has been imagined in science fiction and is usually represented in the form of humanoid robots that can outthink humans. There is currently nothing close to AGI, and we wouldn't know where to start in building it. When AI researchers speak in terms of a computer "understanding" a problem this is a technical definition that describes how a computer has learned a specific task, and does not mean that a computer program has gained the ability to think like a human.
 
-Chatbots can fool us into thinking there is understanding. 
+However, our daily exposure to AI such as chatbots can fool us into thinking there is understanding. 
 
 > “I had not realized ... that extremely short exposures to a relatively simple computer program could induce powerful delusional thinking in quite normal people.”
 >
 > Author of Eliza, the first chatbot
 
 ## Types of AI that DO Exist
 
-AI could be defined as anything done by a computer that would require intelligence if performed by a person. This is a useful but limited definition as it is difficult to properly define intelligence objectively. 
+AI can be defined as anything done by a computer that would require intelligence if performed by a person. This is a useful but limited definition as it is difficult to properly define intelligence objectively. 
 
 ### Simulation
 
-Also known as "running the equations", this may not seem like *real* AI. Simulations allow you to make predictions, which requires some sort of intelligence!
+Also known as "running the equations", this may not seem like *real* AI. Simulations allow you to make predictions, which requires some sort of intelligence! Predicting where the planets will be in the solar system months or years in advance is possible because we know how the solar system works and we can mathematically simulate its movements. To create a simulation, you need to understand how the system you are studying behaves, which allows you to create a model of that system. Today's simulations are more advanced than ever because they use what's known as "digital twinning" to incorporate real-time data to update the model based on the actual behavior of the system being modeled.
```

Comment:
I think we should say "take for example digitial twinning" as there are many situations for simulation work.